setup.py

```diff
@@ -40,19 +40,6 @@ def find_version(*file_paths):
 # https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/_build_utils/openmp_helpers.py
 
 
-class _TemporaryDirectory:
-    def __enter__(self):
-        self.dir_name = tempfile.mkdtemp()
-        return self.dir_name
-
-    def __exit__(self, exc_type, exc_value, traceback):
-        shutil.rmtree(self.dir_name)
-
-
-TemporaryDirectory = getattr(
```

Comment:
You have to import TemporaryDirectory from tempfile now or use it by ``tempfile.TemporaryDirectory``

Reply:
ups, thanks